crates/factor-sqlite/src/lib.rs

```diff
@@ -205,4 +209,11 @@ pub trait Connection: Send + Sync {
     ) -> Result<v2::QueryResult, v2::Error>;
 
     async fn execute_batch(&self, statements: &str) -> anyhow::Result<()>;
+
+    /// A human-readable summary of the connection's configuration
+    ///
+    /// Example: "libSQL at libsql://example.com"
+    fn summary(&self) -> Option<String> {
```

Comment:
What's the thinking around returning `Option` here and defaulting this implementation? Seems like we're just asking for new implementors to forget to implement this method leading to subtle bugs. Why not force implementors to implement this?

Reply:
My thinking is that this is really only applicable to `spin up`, so if some other runtime has its own store impls it probably doesn't care about this method.

Reply:
also the consequence for omitting this is small, at least as currently used